README.md

```diff
@@ -3,154 +3,110 @@ This repository holds the kcp tests that tests against the publicly available  i
 
 ## Prerequisites
 * Git installed. See [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* Golang installed. See [Installing Golang](https://golang.org/doc/install),the newer the better.
-        * Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf`
+* Golang installed. See [Installing Golang](https://golang.org/doc/install), the newer the better ( Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf` ）
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
-
-### Include new test folder
-If you create a new folder for your test case, **add the path** to the [include.go](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/include.go)
-
-### Create go-bindata for new YAML files
-If you have some **new YAML files** used in your code, you have to generate the bindata first.
-Run `make update` to update the bindata. For example, you can see the bindata has been updated after running the `make update` as follows:
-```console
-$ git status
-	modified:   test/extended/testdata/bindata.go
-	new file:   test/extended/testdata/olm/etcd-subscription-manual.yaml
-```
-
-### Compile the executable binary
-Note that we use the `go module` for package management, the previous `go path` is deprecated.
-```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
-$ make build
-mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
-```
-
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git) ( Ensure your installed KCP kubectl plugin version is the same with the kcp service version )
+  - If you would like to download the suitable version packages directly go to [kcp release assets](https://github.com/kcp-dev/kcp/releases), download the zip file suitable for your OS, extract and copy the binaries to a place which is in your PATH and once copied run `kubectl-kcp --version` to verify if server matches with client 
```

Comment:
```suggestion
  - If you would like to download the suitable version packages directly go to [kcp release assets](https://github.com/kcp-dev/kcp/tags), download the zip file suitable for your OS, extract and copy the binaries to a place which is in your PATH and once copied run `kubectl-kcp --version` to verify if server matches with client 
```
It is better to use https://github.com/kcp-dev/kcp/tags. You see it can show a more latest one (currently v0.8.x) while https://github.com/kcp-dev/kcp/releases may show an outdated one (currently v0.7.x). Since this README will be for newcomers, if they follow https://github.com/kcp-dev/kcp/releases and use, they will possibly use an outdated one and then hit issue which is raised in Slack a few times which this suggestion can avoid.

Reply:
Hi @xingxingxia , the https://github.com/kcp-dev/kcp/releases also shows the latest one v0.8.x. Maybe `tags` not better for download as only release versions will build all archs plugins. E.g. https://github.com/kcp-dev/kcp/releases/tag/pkg%2Fapis%2Fv0.7.5 has no plugins assert as it's not a release version but it's in tags list. WDYT?
cc to @kasturinarra

Reply:
OK, I did not scroll down in the page, its top shows v0.7.x ... so I thought ... : )

Reply:
Got it. Thanks!